synfig-studio/src/gui/app.cpp

```diff
@@ -1366,6 +1366,8 @@ App::get_default_accel_map()
 		{"<Mod1>5",                 "<Actions>/canvasview/mask-widthpoint-position-ducks"},
 		{"<Shift>Page_Up",          "<Actions>/action_group_layer_action_manager/action-LayerRaise"},
 		{"<Shift>Page_Down",        "<Actions>/action_group_layer_action_manager/action-LayerLower"},
+		{"Page_Down",               "<Actions>/action_group_layer_action_manager/action-LayerToBottom"},
+		{"Page_Up",                 "<Actions>/action_group_layer_action_manager/action-LayerToTop"},
```

Comment:
I'm not sure if those should be the shortcuts. Or even if these actions should have one.